tests/unit/src/test/scala/tests/DefinitionSlowSuite.scala

```diff
@@ -84,9 +84,9 @@ object DefinitionSlowSuite extends BaseSlowSuite("definition") {
            |import java.util.concurrent.Future/*Future.java*/ // unused
            |import scala.util.Failure/*Try.scala*/ // unused
            |import org.scalatest.FunSuite/*FunSuite.scala*/
-           |object MainSuite/*L4*/ extends FunSuite/*FunSuite.scala*/ {
+           |object MainSuite/*<no symbol>*/ extends FunSuite/*FunSuite.scala*/ {
```

Comment:
Now, the symbol that represents a definition shows the locations of its references, instead of their locations themselves.
In this case, `MainSuite` shows `<no symbol>` because `MainSuite` represents a definition, and no one refers to `MainSuite`.

Reply:
I am inclined to fall back to the original behaviour in this case and return the definition result. The reason is to avoid confusing messages like "No definition found for MainSuite"

![Screenshot 2019-06-18 at 15 00 59](https://user-images.githubusercontent.com/1408093/59684243-e6f11200-91d9-11e9-9684-46697b894b33.png)

Reply:
Sounds reasonable! done https://github.com/scalameta/metals/pull/775/files#diff-f8c05eebbf12c9c21a7d568f09b500eaR1457